pyproject.toml

```diff
@@ -39,7 +39,7 @@ min-public-methods = 1
 [tool.coverage.report]
 fail_under = 100
 show_missing = true
-omit = ["kedro/templates/*", "kedro/framework/cli/hooks/specs.py", "kedro/framework/hooks/specs.py", "kedro/extras/datasets/tensorflow/*"]
+omit = ["kedro/templates/*", "kedro/framework/cli/hooks/specs.py", "kedro/framework/hooks/specs.py", "kedro/extras/datasets/tensorflow/*", "kedro/extras/datasets/holoviews/*", "tests/*"]
```

Comment:
I've added "tests/*" to be omitted, because the coverage was failing for Python 3.10 because `tests_holoviews_writer.py` is skipped. But it made me wonder why the tests weren't omitted in the first place?

Reply:
I think it's because the existing `skipif`s refer to Windows, and in Windows we don't do exactly `make test` - we actually do `make test-no-spark`, which includes `--no-cov`.

Fine by me if we just ignore everything in `tests/*`, but in this case it would be tidy to remove the `--ignore tests/...` options wherever they appear elsewhere (Makefile and CircleCI config I guess).

Reply:
I'm not sure that's correct, `--ignore ...` makes sure not to run the tests, so that should still happen. If we remove `--no-cov` for windows it will fail coverage because certain tests aren't run on windows thus not "covering" that code.

Reply:
I'm also not entirely sure why we run coverage on tests, though _sometimes_ it handily uncovers some overcomplicated fixtures or tests.

Reply:
Oh yes, sorry - forget what I said 🙈